### ISSUE TITLE
https://docs.mybinder.org/ cert is invalid

### ISSUE THREAD

**Issue by betatim** [jupyterhub/mybinder.org-deploy#766]
Firefox tells me:
```
docs.mybinder.org uses an invalid security certificate.
The certificate is not trusted because it is self-signed.
The certificate is only valid for ingress.local. 
Error code: MOZILLA_PKIX_ERROR_SELF_SIGNED_CERT
```

**Comment by betatim** [comment]
We seem to be constantly requesting certificates for our redirect domains: https://crt.sh/?q=%25.mybinder.org

**Comment by minrk** [comment]
This is fixed by #770
